homeassistant/components/spotify/browse_media.py

```diff
@@ -143,6 +145,34 @@ class UnknownMediaType(BrowseError):
     """Unknown media type."""
 
 
+async def async_get_media_browser_root_object(
+    hass: HomeAssistant,
+) -> list[BrowseMedia]:
+    """Create a Spotify root object for media browsing."""
+    return [
+        BrowseMedia(
+            title=f"Spotify {info[DATA_SPOTIFY_ME]['display_name']}",
```

Comment:
This is the Spotify display name, we should maybe use the config entry's title instead?

Reply:
Or the device name?

Reply:
Gave a little more thought: Config entry title makes sense.